src/test/java/org/folio/rest/api/TestBase.java

```diff
@@ -94,4 +100,28 @@ void assertGetNotFound(URL url) {
       throw new RuntimeException(e);
     }
   }
+
+  /**
+   * Map RowSet to JsonArray(s). One array per row.
+   * @param rowSet
+   * @return
+   */
+  List<JsonArray> rowSetToJsonArrays(RowSet<Row> rowSet) {
```

Comment:
Can you add static?
Can you move the static method into a better place, for example test/java/org.folio.rest.support.db.RowSetUtil?

Reply:
ok